packages/compass-databases-navigation/src/database-item.tsx

```diff
@@ -26,10 +26,7 @@ const buttonReset = css({
 });
 
 const expandButton = css({
-  // Not using leafygreen spacing here because none of them allow to align the
-  // button with the search bar content. This probably can go away when we are
-  // rebuilding the search also
-  padding: 7,
```

Comment:
Is this removed intentionally? This does make the icons misaligned and especially when database is selected makes the icon to jammed between elements:

![image](https://user-images.githubusercontent.com/5036933/187656270-a9cb31f5-a5e0-42f3-97bc-5e8ae889d060.png)

Reply:
mmm I have to take a second look, may need to re-add it with that alone didn't align though

Reply:
Looks much better with the padding back